extensions/PhabBugz/lib/Util.pm

```diff
@@ -36,11 +37,18 @@ our @EXPORT = qw(
 
 sub get_revision_by_id {
     my $id = shift;
+    my @result_data = get_revisions_by_ids(qw($id));
+
+    return $result_data[0];
+}
+
+sub get_revisions_by_ids {
+    my ($ids) = @_;
```

Comment:
I'm getting the following error here:

```
Phabricator: Message #8: TRANSIENT-ERROR Code: ERR-CONDUIT-CORE Reason: Error while reading "ids": Expected a
list, but value is not a list.
 at /loader/0x4728448/Bugzilla/Extension/PhabBugz/Util.pm line 270.
	Bugzilla::Extension::PhabBugz::Util::request(...) called at /loader/0x4728448/Bugzilla/Extension/PhabBugz/Util.pm line 48
	Bugzilla::Extension::PhabBugz::Util::get_revisions_by_ids(...) called at ./extensions/Push/lib/Connector/Phabricator.pm line 91
	Bugzilla::Extension::Push::Connector::Phabricator::send(...) called at /loader/0x49a0dc8/Bugzilla/Extension/Push/Push.pm line 129
	eval {...} called at /loader/0x49a0dc8/Bugzilla/Extension/Push/Push.pm line 128
	Bugzilla::Extension::Push::Push::push(...) called at /loader/0x49a0dc8/Bugzilla/Extension/Push/Push.pm line 55
	Bugzilla::Extension::Push::Push::start(...) called at /loader/0x49a0dc8/Bugzilla/Extension/Push/Daemon.pm line 94
	Bugzilla::Extension::Push::Daemon::gd_run(...) called at /usr/local/share/perl5/Daemon/Generic.pm line 192
	Daemon::Generic::new(...) called at /usr/local/share/perl5/Daemon/Generic.pm line 37
	Daemon::Generic::newdaemon(...) called at /loader/0x49a0dc8/Bugzilla/Extension/Push/Daemon.pm line 23
	Bugzilla::Extension::Push::Daemon::start(...) called at extensions/Push/bin/bugzilla-pushd.pl line 22
```

When I $logger->info() what I'm passing to this method, I'm simply seeing `5`; is there a way to typecast to ensure this is always list a list?  Help?